app/views/ex_links/edit.html.haml

```diff
@@ -0,0 +1,9 @@
+.col-md-8.col-md-offset-2
+  %h1 Editing External Link
+
+= render 'form'
+
+.col-md-8.col-md-offset-2
+  %a{ :href => ex_link_path(@ex_link), :title => 'Show', :class => 'btn-u btn-u-green' } Show
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Line is too long. [93/80]